app/src/main/res/layout/activity_topic_revision_test.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename this file to `topic_revision_test_activity`

Reply:
Done